e2e/_suites/fleet/fleet.go

```diff
@@ -476,8 +476,7 @@ func (fts *FleetTestSuite) anAgentIsDeployedToFleetWithInstallerAndFleetServer(i
 	services := []deploy.ServiceRequest{
 		agentService,
 	}
-	env := fts.getProfileEnv()
-	err := fts.deployer.Add(fts.currentContext, deploy.NewServiceRequest(common.FleetProfileName), services, env)
+	err := fts.deployer.Add(fts.currentContext, deploy.NewServiceRequest(common.FleetProfileName), services, common.ProfileEnv)
```

Comment:
I removed the reference to the method, as it was not backported to 7.15 because there is no support to
preconfigured policies.
 
cc/ @adam-stokes